### PR TITLE
Add golangci-lint-langserver

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: installer/install-golangci-lint-langserver.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+"$(dirname "$0")/go_install.sh" github.com/nametake/golangci-lint-langserver@latest
```
Comment (johejo) [inline installer/install-golangci-lint-langserver.sh]: Prebuilt binaries are avaliable. https://github.com/nametake/golangci-lint-langserver/releases

File: installer/install-golangci-lint-langserver.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+"$(dirname "$0")/go_install.sh" github.com/golangci/golangci-lint@latest
```
Comment (johejo) [inline installer/install-golangci-lint-langserver.sh]: Same above https://github.com/golangci/golangci-lint#binary

Comment (mattn) [reply installer/install-golangci-lint-langserver.sh]: golangci-lint-langserver does not support Windows binary.

File: installer/install-golangci-lint-langserver.sh (hunk 1 above)
Comment (mattn) [reply installer/install-golangci-lint-langserver.sh]: Let's merge this and will fix installer when the Windows binary will be released.

**Pull request merged**